doc/lua_api.txt

```diff
@@ -3989,6 +3989,12 @@ Environment access
         * mode = `"quick"`: Clear objects immediately in loaded mapblocks,
                             clear objects in unloaded mapblocks only when the
                             mapblocks are next activated.
+* `minetest.load_area(pos1[, pos2])`
+    * Load the mapblock which contains `pos1`
+      or blocks in the area from `pos1` to `pos2` if `pos2` is specified.
```

Comment:
'or mapblocks'

Reply:
How about this? (needs code escapes inside)
`* Load the mapblock containing pos1 and pos2. pos2 defaults to pos1 if not specified.`

Reply:
SmallJoker, '.. mapblocks containing the volume from pos1 to pos2 ...'